packages/console/src/pages/GetStarted/components/Step/index.tsx

```diff
@@ -65,7 +66,19 @@ const Step = (
       </div>
       {isExpanded && (
         <>
-          <ReactMarkdown className={styles.markdownContent}>{metadata}</ReactMarkdown>
+          <ReactMarkdown
+            className={styles.markdownContent}
+            components={{
+              code: ({ className, children }) => {
+                const [, language] = /language-(\w+)/.exec(className ?? '') ?? [];
```

Comment:
just curious are those languages prefixed className are generated by the ReactMarkDown itself?

Reply:
I think yes. The logic is that if you define a code block in your markdown like this
```
```typescript
blah blah
```
ReactMarkdown will consider it as a `language-typescript` className